docs/source/getting_started/index.rst

```diff
@@ -30,7 +30,7 @@ Starting Iroha Node
 
 Creating a Docker Network
 ^^^^^^^^^^^^^^^^^^^^^^^^^
-To operate, Iroha requires a ``PostgreSQL`` database.
+To operate, Iroha requires a ``PostgreSQL`` database, if you use Iroha with PostgreSQL and not RocksDB.
```

Comment:
```suggestion
To operate, Iroha requires a ``PostgreSQL`` database or ``RocksDB``.
```

Reply:
@kuvaldini  This is not working well within the context. But I added some more and it should be ok now, so please take a look